composer.json

```diff
@@ -20,46 +20,46 @@
         "ext-openssl": "*",
         "guzzlehttp/guzzle": "^7.8",
         "monolog/monolog": "^3",
-        "surfnet/stepup-saml-bundle": "^6.0",
-        "symfony/config": "^5.4|^6.3",
-        "symfony/dependency-injection": "^5.4|^6.3",
-        "symfony/form": "^5.4|^6.3",
-        "symfony/framework-bundle": "^5.4|^6.3",
-        "symfony/http-kernel": "^5.4|^6.3",
-        "symfony/twig-bridge": "^5.4|^6.3",
-        "symfony/validator": "^5.4|^6.3"
+        "surfnet/stepup-saml-bundle": "dev-feature/maintenance_nov-25",
+        "symfony/config": "^6.3|^7.0",
+        "symfony/dependency-injection": "^6.3|^7.0",
+        "symfony/form": "^6.3|^7.0",
+        "symfony/framework-bundle": "^6.3|^7.0",
+        "symfony/http-kernel": "^6.3|^7.0",
+        "symfony/twig-bridge": "^6.3|^7.0",
+        "symfony/validator": "^6.3|^7.0"
     },
     "require-dev": {
         "mockery/mockery": "^1.5",
         "overtrue/phplint": "*",
         "phpmd/phpmd": "^2.13",
-        "phpstan/phpstan": "^1.10",
-        "phpstan/phpstan-symfony": "^1.3",
-        "phpunit/phpunit": "^9.6",
-        "sebastian/phpcpd": "^6.0",
-        "slevomat/coding-standard": "^8.13",
-        "squizlabs/php_codesniffer": "^3.7.1",
-        "symfony/phpunit-bridge": "^5.4|^6.3"
+        "phpstan/phpstan": "^2.1",
+        "phpstan/phpstan-symfony": "^2.0",
+        "phpunit/phpunit": "^11.0.0",
+        "rector/rector": "^2.2",
+        "slevomat/coding-standard": "^8.24",
+        "squizlabs/php_codesniffer": "^4.0",
+        "symfony/phpunit-bridge": "^7.3"
     },
     "scripts": {
         "check": [
```

Comment:
In the SAML library you augmented the scripts with a `check-ci` version. May I suggest to either not do that there, or implement it throughout the bundles/apps? Predictability is quite important imo..